docs/book/src/architecture/controllers/cluster.md

```diff
@@ -5,7 +5,7 @@ The Cluster controller's main responsibilities are:
 * Setting an OwnerReference on the infrastructure object referenced in `Cluster.Spec.InfrastructureRef`.
 * Cleanup of all owned objects so that nothing is dangling after deletion.
 * Keeping the Cluster's status in sync with the infrastructure Cluster's status.
-* Creating a kubeconfig secret for [target clusters](../reference/glossary#Workload cluster).
+* Creating a kubeconfig secret for [target clusters](../reference/glossary#workload-cluster).
```

Comment:
```suggestion
* Creating a kubeconfig secret for [target clusters](../../reference/glossary#workload-cluster).
```

I think you need another parent traversal to get out of `architecture`

Reply:
The link you have is https://deploy-preview-1479--kubernetes-sigs-cluster-api.netlify.com/architecture/reference/glossary.html#workload-cluster, which is a 404. It needs to be https://deploy-preview-1479--kubernetes-sigs-cluster-api.netlify.com/reference/glossary.html#workload-cluster.

Reply:
Thanks. I had to add an `.html` at the end of `glossary` as well.